database/migrations/2019_09_17_042742_add_default_permissions_and_roles.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+use App\Models\Permission;
+use App\Models\Role;
+use App\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class AddDefaultPermissionsAndRoles extends Migration
```

Comment:
Steps for this migration:

1. Get all permissions from old permissions table called `entrust_permissions`
2. Get roles from old `entrust_roles` table
3. Get records from `entrust_permission_role`
4. Get records from `entrust_role_user`

 Now seed old records into new tables

5. Seed all permissions into new table `permissions` by using step-1
6. Seed all roles into new table `roles` by using step-2
7. Seed records into new table `role_has_permissions` by using step-3.  Make sure all new role/permissions id inserted.
8. Seed records into new table `model_has_permissions` by using step-4 where model id is user id and model type `App\User`